docs/appendices/release-notes/unreleased.rst

```diff
@@ -65,3 +65,5 @@ Fixes
 - Fixed an issue that caused primary key lookups to return an empty result
   instead of the row identified by the primary key values, if the primary key
   consists of multiple columns and if one of them is of type ``BOOLEAN``.
+
+- Updated the :ref:`editions` documentation
```

Comment:
We usually don't include documentation changes in the release notes.

Not sure if we want to start doing that. The documentation is not strictly bound to a single release but to the release series, the changes will be reflected as soon as the PR is merged.

Reply:
Ah okay, I did not know since it's my first PR here.  Will remove it. 

Would it help to specify this in the PR checklist?
"Added an entry in CHANGES.txt for user facing changes (except for documentation changes)"